models/profileDiffs.js

```diff
@@ -133,6 +133,30 @@ const fetchProfileDiff = async (profileDiffId) => {
   }
 };
 
+/**
+ * Fetches the unobfuscated profileDiff data of the provided profileDiff Id
+ * @param profileDiffId profileDiffId of the diffs need to be fetched
+ * @returns unobfuscated profileDiff Data
+ */
+const fetchProfileDiffUnobfuscated = async (profileDiffId) => {
```

Comment:
I thought of merging this function with the fetchProfileDiff function, but that would create a security risk. I believe a different function for unobfuscated data is fine.